src/package_Swing/mongoContacts.py

```diff
@@ -75,6 +75,7 @@ def contactBook():
             case _:
                 print("Invalid operation.")
 
+
```

Comment:
try:
    client.admin.command('ping')
    print("Pinged your deployment. You successfully connected to MongoDB!")
except Exception as e:
    print(e)

Line 16:  If MongoDB isn't reachable, after printing the error, the program should stop.